docs/source/code-documentation/cli.rst

```diff
@@ -0,0 +1,8 @@
+.. _cli:
+
+CLI
+===
+
+.. currentmodule:: imap_processing
+
+This is the CLI for running IMAP processing per instrument.
```

Comment:
There was 1 million errors for the autogenerated docs (of course) so I opened an issue: #542